pkg/cmd/server/kubernetes/master.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	kapi "k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/client/record"
+	osclient "github.com/openshift/origin/pkg/client"
```

Comment:
So each controller would have to loop through all HPAs, run the check to see if the given scale resource pointed at Origin or Kube, and then skip it if it pointed at the wrong resource type?  That seems a bit redundant...